docs/advanced/bootloader.md

```diff
@@ -2,14 +2,18 @@
 
 This guide explains how to create a bootloader, how to create a main program to go with the bootloader and how to use this bootloader to perform software updates.
 
+## Porting bootloader
+
+Please follow [the bootloader porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/flash/) for adding a bootloader support for your target.
+
 ## Creating the bootloader
 
 Creating a bootloader is similar to creating a regular application. The only additional step you need is to specify the size of the bootloader as a target override in mbed_app.json in the target field "target.restrict_size":
 
 ```
     "target_overrides": {
         ...
-        "NUCLEO_F429ZI": {
+        "<TARGET_NAME>": {
             "target.restrict_size": "0x20000"
```

Comment:
Might want to make this generic too and provide an example in comments to show the format (hex). Example: "target.restrict_size": "<target_specific_mem_range>" #For nucleo_f429zi use 0x20000